src/main/java/spireMapOverhaul/zones/manasurge/modifiers/common/positive/ToughModifier.java

```diff
@@ -0,0 +1,59 @@
+package spireMapOverhaul.zones.manasurge.modifiers.common.positive;
+
+import basemod.abstracts.AbstractCardModifier;
+import basemod.helpers.TooltipInfo;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.evacipated.cardcrawl.mod.stslib.util.extraicons.ExtraIcons;
+import com.megacrit.cardcrawl.actions.common.GainBlockAction;
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import spireMapOverhaul.util.TexLoader;
+import spireMapOverhaul.zones.manasurge.modifiers.AbstractManaSurgeModifier;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static spireMapOverhaul.SpireAnniversary6Mod.makeImagePath;
+
+public class ToughModifier extends AbstractManaSurgeModifier {
+    private static final ModRarity MOD_RARITY = ModRarity.COMMON_MOD;
+    private static final ModEffect MOD_EFFECT = ModEffect.POSITIVE_MOD;
+
+    private static final Texture ICON = TexLoader.getTexture(makeImagePath("ui/ManaSurge/extraIcons/PositiveEnchantmentIcon.png"));
+    private static final int BLOCK = 5;
```

Comment:
Generally block is considered stronger than damage, so maybe decrease thsi by 1

Reply:
Done.